suite-common/wallet-core/src/discovery/discoveryThunks.ts

```diff
@@ -39,6 +44,31 @@ export const LIMIT = 10;
 export const filterUnavailableNetworks = (
     enabledNetworks: NetworkSymbol[],
     device?: TrezorDevice,
+): Network[] =>
+    Object.values(networks).filter((n: Network) => {
+        const firmwareVersion = getFirmwareVersion(device);
+        const internalModel = device?.features?.internal_model;
+
+        const isSupportedInSuite =
+            !n.support || // support is not defined => is supported
+            !internalModel || // typescript. device undefined. => supported
+            (n.support[internalModel] && // support is defined for current device
+                versionUtils.isNewerOrEqual(firmwareVersion, n.support[internalModel] as string)); // device version is newer or equal to support field in networks => supported
+
+        return (
+            enabledNetworks.includes(n.symbol) &&
+            !n.isHidden &&
+            !device?.unavailableCapabilities?.[n.symbol] && // exclude by network symbol (ex: xrp on T1B1)
+            isSupportedInSuite
+        );
+    });
+
```

Comment:
Because this fn is common for discovery both in `suite` and `suite-native`, and in this PR I want to refactor only native, I copied the old fn :arrow_down:  for `suite`, while the new fn :arrow_up: is used in native (and those cases in this file where it's mapped to symbols, then it doesn't matter :slightly_smiling_face: )